occurrence-download/src/main/java/org/gbif/occurrence/download/file/simplecsv/SimpleCsvArchiveBuilder.java

```diff
@@ -1,5 +1,7 @@
 package org.gbif.occurrence.download.file.simplecsv;
```

Comment:
this class looks that it was incorrectly merged with the master branch since many "final" unnecessary keywords were removed

Reply:
was that ok to remove them ? as they were unnecessary ?